build-settings/conventions/src/main/kotlin/convention-plugins.settings.gradle.kts

```diff
@@ -0,0 +1,10 @@
+pluginManagement {
+    plugins {
```

Comment:
why empty plugin block?

Reply:
oh, I forgot to move here not applied plugins with versions to not write it everywhere